ros_buildfarm/templates/release/rpm/import_upstream_job.xml.em

```diff
@@ -25,7 +25,7 @@
             'type': 'string',
             'name': 'REMOTE_SOURCE_EXPRESSION',
             'description': 'Expression to match pulp remote repositories',
-            'default_value': '^ros-bootstrap-([^-]*-[^-]*-[^-]*(-debug)?)$',
+            'default_value': '^ros-upstream-[^-]*-([^-]*-[^-]*-[^-]*(-debug)?)$',
```

Comment:
I didn't catch it on the previous review but should these zero-or-more `*` be changed to one-or-more `+` in order to avoid matching `ros-upstream----`?
```suggestion
            'default_value': '^ros-upstream-[^-]+-([^-]+-[^-]+-[^-]+(-debug)?)$',
```

Reply:
I often forget about `+` over `*` because I use sed (without -E) so often.

It seems to work fine in Python 3's regular expression. Thanks for the improvement.